lib/meetups_data.rb

```diff
@@ -47,4 +48,79 @@ def self.from_hash(data)
       self.new(data)
     end
   end
+
+  class GroupOrganizer
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def find_organizers
+
+      return if !data["url"].include? 'http://www.meetup.com/'
+
+      return if has_organizers?
+
+      group_urlname = parse_url_name_from_url(data["url"])
+      get_organizers(group_urlname)
+
+    end
+
+    def has_organizers?
+      data.has_key?("organizers")
+    end
+
+    def parse_url_name_from_url(meetup_url)
+      urlname = meetup_url.split('http://www.meetup.com/')
```

Comment:
Makes sense. You should be able to type http://meetup.com or meetup.com and still be able to grab the organizers data.